Amazon.Ion.ObjectMapper/IonObjectSerializer.cs

```diff
@@ -101,8 +152,15 @@ public override void Serialize(IIonWriter writer, object item)
                 ionSerializer.Serialize(writer, propertyValue);
             }
 
+            // Serialize any fields that satisfy the options/attributes.
             foreach (var field in Fields())
             {
+                if (serializedWithGetter.Contains(field.Name))
```

Comment:
I am unsure this is the right behaviour.

Reply:
Likewise we are not gonna serialize the field if the name was already serialized via a method.